config/clusters/2i2c/ohw.values.yaml

```diff
@@ -74,6 +74,8 @@ basehub:
         add_staff_user_ids_to_admin_users: true
         add_staff_user_ids_of_type: "github"
       homepage:
+        hubTemplateRepoDetails:
+          repo_branch: "2i2c-ohw"
         templateVars:
           org:
             name: OceanHackWeek
```

Comment:
I figure `custom.homepage` is already implying `custom hub template` since there is `custom.homepage.templateVars` already.

What do you think about a structure like this?

```yaml
custom:
  homepage:
    gitRepoUrl: ...
    gitRepoBranch: ...
    templateVars: ...
```

I switched to camelCase for consistency and the practice of using such syntax for what's pure chart config, added `git` prefix for what I think could be helpful verbosity.

Reply:
Note that to de-nest this to be flat - directly under custom.homepage is aligning with advice in helm chart docs: https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values

> In most cases, flat should be favored over nested. The reason for this is that it is simpler for template developers and users.

Reply:
> I figure custom.homepage is already implying custom hub template since there is custom.homepage.templateVars already.

I like the proposed removal of the `hubTemplateRepoDetails` details. Thank you for thinking and explaining the reasoning behind it <3! In my head I couldn't reason why they are supposed to be on the same tree level, but adding the `git` prefix in the key name does a great job grouping those together and explaing it.